media/doc/WINESYNC.txt

```diff
@@ -206,7 +206,7 @@ dll/win32/version             # Synced to WineStaging-4.18
 dll/win32/vssapi              # Synced to WineStaging-4.18
 dll/win32/wbemdisp            # Synced to WineStaging-4.18
 dll/win32/wbemprox            # Synced to WineStaging-4.18
-dll/win32/windowscodecs       # Synced to WineStaging-4.18
+dll/win32/windowscodecs       # Synced to Wine-10.0
 dll/win32/windowscodecsext    # Synced to WineStaging-2.9
```

Comment:
Maybe this one could be synced as well (from what I see in wine-mirror that dll is mostly a stub).